lib/tasks/default.rake

```diff
@@ -427,6 +427,62 @@ if Rails.env.development?
   end
 end
 
+require 'open-uri'
+# Returns "true" if okay
+# rubocop:disable Metrics/MethodLength
+def validate_one_link(link)
```

Comment:
Ruby methods that return a boolean should end in `?`. So I recommend:
```ruby
def single_link_returns_success?(link)
```
And then the comment `# Returns "true" if okay` is unnecessary.